README.md

```diff
@@ -8,7 +8,7 @@ Beam is currently in a developer preview, and both the service and the documenta
 
 ### TL;DR
 
-If you would like to get your hands dirty without reading our documentation, we strongly advise you to get started with one of our implementations ready [SDKs](https://docs.onbeam.com/service/sdk). We currently provide SDKs for Node.js and C#, but are happy to facilitate more. If you are interested in an SDK for your implementation language which we don't provide yet, please open an [issue](https://github.com/Merit-Circle/beam-docs/issues/new).
+If you would like to get your hands dirty without reading our documentation, we strongly advise you to get started with one of our implementations ready [SDKs](https://docs.onbeam.com/service/sdk). We currently provide SDKs for Node.js and C#, but are happy to facilitate more. If you are interested in an SDK for your implementation language which we don't provide yet, please open an [issue](https://github.com/Merit-Circle/beam-sdk/issues/new).
```

Comment:
@lewinskimaciej  @RobbyUitbeijerse changed the link at the end to open an issue on the SDK repo instead of the Docs repo.